my-code/movie-search/src/App.css

```diff
@@ -0,0 +1,159 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500,700');
+
+body {
+  background: #0A1014;
+  font-size: 16px;
+  font-family: 'Roboto', sans-serif;
+  color: #7A8C99;
+}
+
+h1 {
+  font-size: 64px;
+  margin: 30px 0;
+}
+
+h3 {
+  font-size: 24px;
+}
+
+h4 {
+  font-size: 20px;
+  font-weight: 700;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.App {
+  display: grid;
+  grid-template-columns: [full-start] 130px [content-start] auto [content-end] 130px [full-end];
+}
+
+.LandingPage, .MovieDetails {
+  grid-column-start: content-start;
+  grid-column-end: content-end;
+}
+
+.AppLogo {
+  padding: 25px 0 15px;
+  grid-column-start: content-start;
+}
+
+.SearchBar input {
+  width: 100%;
+  border: none;
+  border-radius: 5px;
+  padding: 12px 12px 12px 16px;
+  font-size: 16px;
+  font-weight: 300;
+  color: #0A1014;
+}
+
+::-webkit-input-placeholder { /* Chrome */
+  color: #7A8C99;
+}
+:-ms-input-placeholder { /* IE 10+ */
+  color: #7A8C99;
+}
+::-moz-placeholder { /* Firefox 19+ */
+  color: #7A8C99;
+  opacity: 1;
+}
+:-moz-placeholder { /* Firefox 4 - 18 */
+  color: #7A8C99;
+  opacity: 1;
+}
+
+.MovieCatalogue ul {
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
```

Comment:
List style could be solved at the Reset file. It would make CSS rules to the point and cleaner